Description: migrate to PNG 1.6
 Migrate from PNG 1.2 to PNG 1.6
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Last-Update: 2020-05-01
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/PNGImage.cpp
+++ b/src/PNGImage.cpp
@@ -53,7 +53,7 @@
 	}
 
 	// use libpngs default error handling
-	if (setjmp(png_ptr->jmpbuf))
+	if (setjmp(png_jmpbuf(png_ptr)))
 		cerr << "PNG: setjmp() failed\n";
 
 	png_init_io(png_ptr, fp);
